x/crosschain/keeper/cctx_orchestrator.go

```diff
@@ -169,20 +193,58 @@ func (k Keeper) processFailedOutboundForExternalChainTx(
 		if err != nil {
 			return err
 		}
+		if revertMsg == "" {
```

Comment:
why not just send this string when calling `validateFailedOutbound` from `validateFailedOutboundObservers` , so that we don't need this check